multi-repo-action/scorecards.yml

```diff
@@ -0,0 +1,58 @@
+name: Scorecards supply-chain security
```

Comment:
We need to keep this workflow up-to-date, in particular the hashes. Ideally we should use the hash for the latest version of scorecard and set it dynamically to `ossf/scorecard-action@hash`. Try thinking of a way to do this. Otherwise it's yet another file to keep up-to-date manually.

Maybe to keep things simple: we could update the release process in https://github.com/ossf/scorecard-action/issues/33. The only downside of this is that if a user clones the repo, they won't get the update unless they `git pull`.

Note that an out-of-date hash is not a super big deal in practice: dependabot will notice the hash is old and create a PR to update it. So I think updating the https://github.com/ossf/scorecard-action/issues/33 is the way to do.

So I just ignore everything I said :-)

Reply:
Made it so the latest workflow file from https://github.com/ossf/scorecard/blob/main/.github/workflows/scorecard-analysis.yml is pulled

Reply:
maybe we could use a workflow template committed to this repo, and use ascii doc to include the template into the README. The scorecard workflow has lots of comments tat we should avoid in the PR.

Reply:
So the scorecard-analysis.yml file that gets created should have no comments, and the original version gets put in the README? The yml doesn't seem to have that many comments though.

Reply:
the scorecard yaml from scorecard repo is generally not up-to0date: it gets alerts from dependabot to be updated when we release a new action. It's cleaner to keep it in this repo

Reply:
Something that I just thought of. The workflow needs its own PAT to run, which is usually defined as an env secret. Should we just tell the users to manually create an org-level secret that the workflow files will pick up? If we want to automate it then the script will need higher permissions. @azeemshaikh38 @laurentsimon

Reply:
I mean the multi-repo-action script will need elevated permissions if we want it to create the secret for the org. Right now it just has basic read permissions, so I think we'd want to avoid that route. 

Then, for the workflow to run after being uploaded, the org owner will have to manually set a secrets.SCORECARD_READ_TOKEN secret on Github. 

The PAT defined in this script is just so it can read repos and make the PRs. The actual workflow will need to pull its own token from an org secret.

Reply:
got it. Agreed it's not ideal to have the PAT with write permission. My original idea of using workflow seems to be a bad one. Maybe we should just create a docker image and let users run it and provide their PAT on the CLI. wdut?

The PAT can be used to create the secrets (if needed for scorecard to run - this may change), and for the pull request creation. wdut?

Reply:
That would be a good solution. I don't think the convenience is drastically different, so maybe we can save this issue for a future release and keep the current approach as a v0?